src/frontends/leaflet.ts

```diff
@@ -43,8 +43,8 @@ const reflect = (promise: Promise<Status>) => {
 type DoneCallback = (error?: Error, tile?: HTMLElement) => void;
 type KeyedHtmlCanvasElement = HTMLCanvasElement & { key: string };
 
-interface LeafletLayerOptions {
-  bounds?: number[][];
+export interface LeafletLayerOptions extends L.GridLayerOptions {
+  bounds?: L.LatLngBoundsExpression;
```

Comment:
On 2nd thought, `bounds?: L.LatLngBoundsExpression;` may not be necessary at all if this is now extending from `L.GridLayerOptions` in the first place. But leaving it here reinforces for readability's sake which options this custom layer deals with itself during construction.

Reply:
I also added the extension because I was doing things like successfully passing in `minZoom`, which is defined on the `L.GridLayerOptions`.